### PR TITLE
Some more missing newlines fixed

### PR DESCRIPTION
**Pull request by andreas-roehler**

### CODE REVIEW

File: tuareg.el [hunk 1]
```diff
@@ -72,6 +72,7 @@
 
 ;;; Code:
 
+
```
Comment (Chris00) [inline tuareg.el]: ?

**Comment by Chris00** [comment]
Could you please *rebase* on master (there should be no “merge commit”.

**Comment by andreas-roehler** [comment]
Called rebase locally. Don't know how to replace the branch at github, as being told: "Everything up-to-date" when push -f. Maybe collapse the history later?

File: tuareg.el (hunk 1 above)
Comment (andreas-roehler) [inline tuareg.el]: Just a missing call to my-just-one-line afterwards. Will fix this with next commit.

**Comment by Chris00** [comment]
Did you do `git rebase origin/master` (or whatever this repo is called instead of `master`)?

**Comment by Chris00** [comment]
I [cherry-picked](https://github.com/ocaml/tuareg/commit/0920d003008132ce371b76616a8ab33fb00c7b70) the commit.

**Comment by andreas-roehler** [comment]
Thanks! Will try my best :)

**Pull request closed without merging**